docs/user-guide/cli-using-using-daemon-mode.md

```diff
@@ -1,23 +1,24 @@
+:::info Required roles: Security administrator, DevOps architect
+:::
 
-**Important:** We do not recommend using daemon mode in an environment where multiple users use the same system. For example, a shared Linux server.
+When you run Zowe CLI in daemon mode, you run all Zowe commands as you normally run them. The first time you run a command, it starts the daemon in the background automatically and runs your desired Zowe command. Since the first Zowe command starts the daemon, the first command usually runs slower than a traditional Zowe command. However, subsequent Zowe commands run significantly faster. The daemon continues to run in the background until you close your terminal window.
 
 ## Preparing for installation
 
 Review the following installation notes before you configure Zowe CLI to run in daemon mode:
 
 - Daemon mode does not function on z/OS UNIX System Services (USS) systems.
-- When you want to run Zowe CLI to run in daemon mode on **z/Linux** operating systems, you must build the daemon mode binary on the z/Linux systems. For information about how to build the binary, see [Configure Secure Credential Store on headless Linux operating systems](cli-configure-scs-on-headless-linux-os.md). The sections [Enable daemon mode](#enable-daemon-mode) and [Disable daemon mode](#disable-daemon-mode) (in this article) **do not apply** to running Zowe CLI in daemon mode on z/Linux operating systems.
-- We do not recommend using daemon mode in an environment where multiple users use the same system. For example, a shared Linux server.
-- When you are running Zowe on a Windows operating system in a virtual environment (for example, Windows Sandbox), you might receive an error message that indicates that a library named `VCRUNTIME140.dll` is missing. To correct the error, install Visual C++ Redistributable for Visual Studio 2015. For more information, see [Download Visual C++ Redistributable for Visual Studio 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48145). 
+- When you want Zowe CLI to run in daemon mode on z/Linux operating systems, you must build the daemon mode binary on the z/Linux systems. For information about how to build the binary, see [Configuring daemon mode on z/Linux operating systems](../user-guide/cli-configure-daemon-on-zlinux-os.md). The sections [Enabling daemon mode](#enabling-daemon-mode) and [Disabling daemon mode](#disabling-daemon-mode) (in this article) **do not apply** to running Zowe CLI in daemon mode on z/Linux operating systems.
+- We do not recommend using daemon mode in an environment where multiple users use the same system. For example, a shared Linux server. **[can we explain why we don't recommend this?]**
```

Comment:
@adam-wolfe @zFernand0 - please respond

Reply:
This may be an artifact from when we used TCP/IP as the communication mechanism. I think we support a multi-tenant environment, but each person would have their own Zowe daemon (which may result in increased resource consumption).